tests/containers/setup_centos.pm

```diff
@@ -0,0 +1,25 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Setup Centos
+# - setup networking via dhclient
+# - import SUSE CA certificates
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base qw(consoletest);
+use testapi;
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+    assert_script_run "dhclient -v";
```

Comment:
I agree in having modules to setup specific Host OS, but in this case it's only to run 1 command... I wonder if it would be more efficient moving this command to `install_docker_when_needed.pm` in the condition for centos.

Reply:
i dont like it too. But also it does not belong to other places. So one idea was to rename the module to reflect to the general setup. As this will possible include configuration for ubuntu and alpine later. But also i could merge it with the `import_ca_certs.pm`

Reply:
Up to you, I don't mind if we have a module for each 3rd party host setup. Let's see what others think.

Reply:
i made some changes here af17ee2de